assets/css/guides.less

```diff
@@ -1091,6 +1091,17 @@ body .guides .guides-section-white {
       background: transparent !important;
     }
   }
+
+  h3#what-youll-learn-in-this-guide ~ .dlist {
```

Comment:
If you're already target an ID, why bother with the `h3`?

And what does the `~` selector do?

Also, should _all_ links in a definition list be styled this way? Or _only_ the ones in "what you'll learn in this guide" section?

Reply:
Taken off the h3. 
~ is a sibling selector which I used to select `.dlist` class
Only what is within the `what you'll learn in this guide` section needs to be styled this way; that's why I'm selecting the id of that section

Reply:
Roger